tutorials/tracing/llama_index_openai_agent_tracing_tutorial.ipynb

```diff
@@ -2,7 +2,11 @@
  "cells": [
```

Comment:
<p>Is it worth requiring the pandas import just for that column width? I have run the colab with and without it and there is barely a difference</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1466/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>